src/mongo/db/pipeline/expression_test.cpp

```diff
@@ -2864,6 +2864,24 @@ TEST(ExpressionObjectOptimizations, OptimizingAnObjectShouldOptimizeSubExpressio
     ASSERT_VALUE_EQ(expConstant->evaluate(Document()), Value(3));
 };
 
+TEST(ExpressionObjectOptimizations, OptimizingAnObjectWithAllConstantsShouldOptimizeToExpressionConstant) {
+
+    intrusive_ptr<ExpressionContextForTest> expCtx(new ExpressionContextForTest());
+    VariablesParseState vps = expCtx->variablesParseState;
+
+    //All constants should optimize to ExpressionConstant
+    auto object1 =  ExpressionObject::parse(expCtx, BSON("b" << 1 << "c" << 1),vps);
```

Comment:
Don't use numbers in your variable names. If these tests are different, split them out into separate tests, otherwise, feel free to re-use the variable names and replace their values.